apps/cowswap-frontend/src/modules/ordersTable/pure/ReceiptModal/FeeField.tsx

```diff
@@ -7,23 +7,42 @@ import * as styledEl from './styled'
 
 export type Props = { order: ParsedOrder }
 
+// TODO: get rid of this once https://github.com/cowprotocol/services/pull/3184 is complete
+const HAS_BACKEND_MIGRATED = false
+
+function getFeeToken(order: ParsedOrder) {
+  if (!HAS_BACKEND_MIGRATED) {
+    return order.inputToken
+  }
```

Comment:
Until a migration is done, the fee will continue being returned in SELL token

https://cowservices.slack.com/archives/C036G0J90BU/p1735581010053059?thread_ts=1735059402.066139&cid=C036G0J90BU

Reply:
Actually, decided to default to `inputToken` when the feeToken is not set.
This flag is not needed then.